tests/dem/insertion_non_uniform.cc

```diff
@@ -56,27 +56,35 @@ test()
   DEMSolverParameters<dim> dem_parameters;
 
   // Defining simulation general parameters
-  const unsigned int n_properties                     = 21;
-  dem_parameters.insertion_info.x_min                 = -0.05;
-  dem_parameters.insertion_info.y_min                 = -0.05;
-  dem_parameters.insertion_info.z_min                 = -0.05;
-  dem_parameters.insertion_info.x_max                 = 0.05;
-  dem_parameters.insertion_info.y_max                 = 0.05;
-  dem_parameters.insertion_info.z_max                 = 0.05;
-  dem_parameters.insertion_info.inserted_this_step    = 10;
-  dem_parameters.insertion_info.distance_threshold    = 2;
-  dem_parameters.physical_properties.diameter         = 0.005;
-  dem_parameters.physical_properties.density          = 2500;
-  dem_parameters.insertion_info.total_particle_number = 10;
-  dem_parameters.insertion_info.random_number_range   = 0.75;
-  dem_parameters.insertion_info.random_number_seed    = 19;
+  const unsigned int n_properties                                 = 21;
```

Comment:
Why not get this object from the DEM properties enum? Why hardcode it here?

Reply:
If this is other type of properties than particle properties, then give it another name (and add comments).